modules/alpha_fold/README.md

```diff
@@ -1,18 +1,33 @@
-# RAD Lab Alpha Fold Module
+# RAD Lab alpha fold module
 
-This RAD Lab module utilizes [Data Science Module](../data_science) and deploy the [AlphaFold](https://cloud.google.com/blog/products/ai-machine-learning/running-alphafold-on-vertexai) configutaions over Vertex AI Workbench.
+RAD Lab enables users to deploy infrastructure on Google Cloud Platform (GCP) to support specific use cases. Infrastructure is created and managed through [Terraform](https://www.terraform.io/) in conjunction with support scripts written in Python. 
```

Comment:
> RAD Lab enables users to deploy infrastructure on Google Cloud Platform (GCP) to support specific use cases. Infrastructure is created and managed through [Terraform](https://www.terraform.io/) in conjunction with support scripts written in Python.

This description is already covered in the RAD Lab Repo's About section and can be skipped. May be add more details on what this RAD Lab module will comprise off. 

You can have something like:

> Bio-pharma organizations can now leverage the groundbreaking protein folding system, AlphaFold, with Vertex AI and this RAD Lab module spins up the complete end to end environment for the same.